source/pkcore/abilitypages/chi/swiftkick.rst

```diff
@@ -0,0 +1,63 @@
+.. swiftkick:
+############
+SwiftKick
+############
+
+Description
+###########
+
+SwiftKick is a Chi ability.
+
+SwiftKick allows a chiblocker to swiftly kick an enemy, potentially blocking their chi. GIF TO BE ADDED
```

Comment:
Change to `SwiftKick allows Chiblockers to hit enemies with a powerful kick, dealing increased damage and resulting in a greater chance to block their chi. GIF TO BE ADDED`